pkgs/servers/sql/postgresql/ext/apache_datasketches.nix

```diff
@@ -15,20 +15,20 @@ let
     name = "datasketches-postgresql";
     owner = "apache";
     repo = "datasketches-postgresql";
-    rev = "refs/tags/${version}";
+    tag = version;
     hash = "sha256-W41uAs3W4V7c9O/wBw3rut65bcmY8EdQS1/tPszMGqA=";
   };
 
   cpp_src = fetchFromGitHub {
     name = "datasketches-cpp";
     owner = "apache";
     repo = "datasketches-cpp";
-    rev = "refs/tags/5.0.2";
```

Comment:
It looks like release 1.7.0 might need to stick to version 5.0 of the cpp module, the upcoming 1.8.0 release is the one that got upgraded to 5.2.